benchmarks/buiter_et_al_2016_jsg/exp_1.prm

```diff
@@ -0,0 +1,183 @@
+  end
+end
+
+# Material model
+subsection Material model
+  set Material averaging = harmonic average 
+  set Model name = visco plastic
+  subsection Visco Plastic
+#						      [bkd, sand,  bound, block, sticky_air, total_strain]
+    set Densities                                   = 1560, 1560., 3100,  1560 , 0,          1560
+    
+    set Reference temperature                       = 293
+    set Heat capacities                             = 750.
+    set Thermal diffusivities                       = 1e-6
+    set Thermal expansivities                       = 0.
+    
+    set Reference viscosity                         = 1e8
+    set Minimum viscosity                           = 1e4
+    set Maximum viscosity                           = 1e12
+
+    set Viscosity averaging scheme                  = maximum composition
+    set Viscous flow law                            = dislocation
+    set Prefactors for dislocation creep            = 5e-11,5e-50, 5e-50, 5e-11, 5e-5,       1e5
+    set Stress exponents for dislocation creep      = 1.0
+    set Activation energies for dislocation creep   = 0.
+    set Activation volumes for dislocation creep    = 0.
+ 
+#                                                     [bkd, sand,  bound, block, sticky_air, total_strain]
+    set Reference strain rate                       = 1.e-5
+    set Minimum strain rate                         = 1.e-10
+    set Use strain weakening                        = true
+    set Start plasticity strain weakening intervals = 0,    0.50,  0.5,   0,     0,          1e5
+    set End plasticity strain weakening intervals   = 0.1,  1.00,  1,     0.1,   0.1,        1e4
+
+    set Angles of internal friction                 = 36.,  36.,   1.6,   0.,    0.,         1e5
+    set Friction strain weakening factors           = 1,    0.861, 1,     1,     1,          1e5
+    set Cohesions                                   = 30.,  30.,   3.,    1.e18, 1.e18,      1e5
+    set Cohesion strain weakening factors           = 1,    1.00,  1,     1,     1,          1e5
+
+  end
+end
+
+# Gravity model
+subsection Gravity model
+  set Model name  = vertical
+  subsection Vertical
+    set Magnitude = 9.81
+  end
+end
+
+# Checkpointing
+subsection Checkpointing
+  set Steps between checkpoint = 10
+end
+
+# Post processing
+subsection Postprocess
+  set List of postprocessors = velocity statistics, basic statistics, visualization
+  subsection Visualization
+    set List of output variables      = density, viscosity, strain rate, named additional outputs
+    set Output format                 = vtu
+    set Time between graphical output = 144
+    set Interpolate output            = false
+  end
+end
+
+# Termination Criteria
+subsection Termination criteria
+  set Termination criteria = end step
+  set End step             = 10000
+end
+
+subsection Solver parameters
+  subsection Stokes solver parameters
+    set Linear solver tolerance             = 1e-8
```

Comment:
The non-linear solver tolerance is higher than the linear solver tolerance. My suggestion would be to keep this value, but reduce the non-linear solver to between 1-e4 and 1e-7 (we should talk with @cedrict about this).